less/index.less

```diff
@@ -139,38 +157,72 @@ header .logo {
    border-bottom: 2px solid black;
    margin: 75px 0 35px 0;
    padding: 35px 0;
-   font-size: 16px;
+   font-size: @primary-body-size;
    line-height: 18px;
-}
 
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
-}
+   h4 {
+    font-size: @primary-sub-heading-size;
+   }
 
-.main-content .middle-img {
-    display: block;
-    margin: 30px auto;
-}
+   .top-content {
```

Comment:
Since `top-content` and `bottom-content` utilize the same two lines of code, you could use a mixin here.

Reply:
ah I see it